CliClient/locales/it_IT.po

```diff
@@ -3305,212 +2970,3 @@ msgstr "Accedi a OneDrive"
-#~ "\n"
-#~ "Ad esempio, per creare un taccuino, premere `mb`; per creare una nota "
-#~ "premere `mn`."
-#~ msgstr ""
-#~ "Benvenuto su Joplin!\n"
-#~ "\n"
-#~ "Digita `:help shortcuts` per l'elenco delle scorciatoie da tastiera, o "
-#~ "semplicemente `:help` per informazioni sull'utilizzo.\n"
-#~ "\n"
-#~ "Ad esempio, per creare un taccuino, premere `mb`; per creare una nota "
-#~ "premere `mn`."
-
-#~ msgid "Searches"
-#~ msgstr "Ricerche"
-
-#~ msgid "Imports an Evernote notebook file (.enex file)."
-#~ msgstr "Importa un file taccuino di Evernote (.enex file)."
-
-#~ msgid ""
-#~ "File \"%s\" will be imported into existing notebook \"%s\". Continue?"
-#~ msgstr ""
-#~ "Il file \"%s\" sarà importato nel taccuino esistente \"%s\". Continuare?"
-
-#~ msgid ""
-#~ "New notebook \"%s\" will be created and file \"%s\" will be imported into "
-#~ "it. Continue?"
-#~ msgstr ""
-#~ "Un nuovo taccuino \"%s\" sarà creato e al suo interno verrà importato il "
-#~ "file \"%s\" . Continuare?"
-
-#~ msgid "Import Evernote notes"
-#~ msgstr "Importa le note da Evernote"
-
-#~ msgid "Give focus to next pane"
-#~ msgstr "Pannello successivo"
-
-#~ msgid "Give focus to previous pane"
-#~ msgstr "Pannello precedente"
-
-#~ msgid "Exit command line mode"
-#~ msgstr "Esci dalla modalità linea di comando"
-
-#~ msgid "Edit the selected note"
-#~ msgstr "Modifica la nota selezionata"
-
-#~ msgid "Cancel the current command."
-#~ msgstr "Annulla il comando corrente."
-
-#~ msgid "Set a to-do as completed / not completed"
-#~ msgstr "Imposta un \"Cose-da-fare\" come completato / non completato"
-
-#~ msgid "[t]oggle [c]onsole between maximized/minimized/hidden/visible."
-#~ msgstr ""
-#~ "Scegli lo  s[t]ato della [c]onsole: massimizzato/minimizzato/nascosto/"
-#~ "visibile."
-
-#~ msgid "[t]oggle note [m]etadata."
-#~ msgstr "mos[t]ra/nascondi i [m]etadata nelle note."
-
-#~ msgid "[M]ake a new [n]ote"
-#~ msgstr "Crea ([M]ake) una nuova [n]ota"
-
-#~ msgid "[M]ake a new [t]odo"
-#~ msgstr "Crea ([M]ake) una nuova at[t]ività"
-
-#~ msgid "[M]ake a new note[b]ook"
-#~ msgstr "Crea ([M]ake) un nuovo [b]locco note"
-
-#~ msgid "Copy ([Y]ank) the [n]ote to a notebook."
-#~ msgstr "Copia ([Y]) la [n]ota in un taccuino."
-
-#~ msgid "Move the note to a notebook."
-#~ msgstr "Sposta la nota in un taccuino."
-
-#~ msgid ""
-#~ "The target to synchonise to. If synchronising with the file system, set "
-#~ "`sync.2.path` to specify the target directory."
-#~ msgstr ""
-#~ "La destinazione della sincronizzazione. Se si sincronizza con il file "
-#~ "system, impostare ' Sync. 2. Path ' per specificare la directory di "
-#~ "destinazione."
-
-#~ msgid "To-do title:"
-#~ msgstr "Titolo del \"Cose-da-fare\":"
-
-#~ msgid "\"%s\": \"%s\""
-#~ msgstr "\"%s\": \"%s\""
-
-#~ msgid "Delete notebook?"
-#~ msgstr "Eliminare il taccuino?"
-
-#~ msgid "File system synchronisation target directory"
-#~ msgstr "Cartella di destinazione per la sincronizzazione nel file system"
```

Comment:
As mentioned before: Do not remove these commented translations.

Reply:
Hello. Are you still there?

Reply:
Sorry, I'm going to delete the branches now.

Just give me a motent to find them.

Reply:
No worries For this PR, you only have to add the deleted lines again. That's all.

For the future you will have to create new branches and start from a point that is HEAD at git master.